goji/commands.py

```diff
@@ -114,7 +114,7 @@ def show(client, issue_key):
 
     if issue.description:
         for line in issue.description.splitlines():
-            click.echo('  {}'.format(line))
+            click.echo(u'  {}'.format(line))
```

Comment:
Various other fields in this file will suffer with the same problem for Python 2. I think a better solution would be to place `from __future__ import unicode_literals` at the start so every string literal is unicode.

That way it will apply to assignee, creator, comment, summary etc.